src/reducers/entities/teams.js

```diff
@@ -51,6 +51,14 @@ function teams(state = {}, action) {
 }
 
 function myMembers(state = {}, action) {
+    function updateState(receivedTeams = {}, currenState = {}) {
```

Comment:
Typo here `currenState` ->`currentState` then it looks good

This function's also a bit too much abstraction in my opinion but 0/5